tests/test_particle_sets.py

```diff
@@ -67,8 +67,10 @@ def test_pset_create_fromparticlefile(fieldset, mode, restart, tmpdir):
 
     class TestParticle(ptype[mode]):
         p = Variable('p', np.float32, initial=0.33)
+        p2 = Variable('p2', np.float32, initial=1, to_write=False)
+        p3 = Variable('p3', np.float32, to_write='once')
```

Comment:
Adding this in the TestParticle is good, but down the test function there's no check if `p2` is not written and `p3` is only there once.